assets/js/src/newsletters/listings/mixins.jsx

```diff
@@ -131,7 +131,7 @@ const QueueMixin = {
 
     let progressBarWidth = 0;
 
-    if (isNaN(percentage)) {
+    if (percentage) {
```

Comment:
I don't understand why replacing `isNaN(percentage)` with `percentage` would work here!

Reply:
I think it was a mistake before. `percentage` is defined as:
```
   let percentage = Math.round(
      (newsletter.queue.count_processed * 100) / (newsletter.queue.count_total)
    );
```

And that would not be  nan even if `count_total` is 0

Reply:
But let me double check that.

Reply:
I fixed that, you are right.